frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx

```diff
@@ -1,6 +1,7 @@
 import styled from "@emotion/styled";
 import Button from "metabase/core/components/Button";
 import { color } from "metabase/lib/colors";
+import { space } from "metabase/styled-components/theme";
```

Comment:
Think we don't need that import anymore?

Reply:
Oh snap! good catch.